### ISSUE TITLE
Website Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (a:nth-child(1))

### ISSUE THREAD

**Issue by A11Ybot-OP** [matrix-org/matrix.org#1187]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Matrix.org](https://matrix.org/)
#### Element path
.mxgrid__item50:nth-child(2) > blockquote > p[dir="ltr"][lang="en"] > a:nth-child(1)
#### Snippet
`<a href="https://twitter.com/RiotChat?ref_src=twsrc%5Etfw">@RiotChat</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.25 (foreground color: #0098d4, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62
